pages/_.vue

```diff
@@ -1,6 +1,14 @@
 <template>
   <div>
-    <HeroBanner />
+    <HeroBanner
+      v-if="page.primaryImageOfPage"
+      :hero-image="page.primaryImageOfPage.fields.image.fields.file.url"
+      :headline="page.primaryImageOfPage.fields.headline"
+      :description="page.primaryImageOfPage.fields.description"
+      :identifier="page.primaryImageOfPage.fields.identifier"
```

Comment:
The record identifier coming from Contentful will start with a slash, resulting in two slashes in the record URL link from the hero banner. This needs to be taken into account somewhere.

Reply:
It's in HeroBanner.vue, similar to how this is done on the cards.